src/AutoMapper/Mappers/DynamicMappers.cs

```diff
@@ -21,6 +21,7 @@ public class FromDynamicMapper : IObjectMapExpression
                 destination = (TDestination) (!context.ConfigurationProvider.AllowNullDestinationValues
```

Comment:
Yes, I don't have VS handy right now, but I saw the same code in many of the mappers I've changed. Maybe that code duplication thing in VS would help.